www/front_src/src/PollerWizard/PollerAtoms.ts

```diff
@@ -0,0 +1,40 @@
+import { atom } from 'jotai';
+
+interface PollerData {
+  centreon_central_ip?: string;
+  linked_remote_master?: string;
+  linked_remote_slaves?: Array<string>;
+  open_broker_flow?: boolean;
+  server_ip?: string;
+  server_name?: string;
+  server_type?: string;
+  submitStatus?: boolean;
+}
+
+interface RemoteServerData {
+  centreon_central_ip?: string;
+  centreon_folder?: string;
+  db_password?: string;
+  db_user?: string;
+  // linked_pollers?: Array<string>;
```

Comment:
I assume this is used

Reply:
no it not used

Reply:
At least, make it optional and let it undefined